ArduCopter/flight_mode.cpp

```diff
@@ -6,89 +6,87 @@
  */
 
 // return the static controller object corresponding to supplied mode
-Copter::FlightMode *Copter::flightmode_for_mode(const uint8_t mode)
+Copter::Mode *Copter::mode_for_mode(const uint8_t mode)
```

Comment:
Aahhh... I think we need to do better here :-)   Maybe "mode_for_modenumber"?

Reply:
haha.  that is indeed terrible.  I didn't even notice, it's just the natural outcome of a search and replace.  In Rover it seems we call the equivalent method, "control_mode_from_num" so I'd say we should name them the same thing.. whatever that name is.  "mode_from_num", "mode_from_mode_num" or "mode_from_number" works for me.  Other suggestions also welcome.

Reply:
My preference goes to *mode_from_num*.